nav2_mppi_controller/include/nav2_mppi_controller/critic_manager.hpp

```diff
@@ -46,6 +46,7 @@ class CriticManager
     * @brief Constructor for mppi::CriticManager
     */
   CriticManager() = default;
+  ~CriticManager() = default;
```

Comment:
Does the compiler not simply do this for us? But no matter, we can accept this, no problem

Reply:
It did not in my case. 
I am running a pretty standard setup and just have done a fresh install of Ubuntu 22.04 and ROS2 humble.
So that made me think that this might be a pretty common problem.